server/athenian/api/openapi/openapi.yaml

```diff
@@ -1082,6 +1082,12 @@ paths:
               schema:
                 $ref: '#/components/schemas/Contributors'
           description: List of all contributors belonging to the specified account.
+        403:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/GenericError'
+          description: The user is forbidden to list contributors belonging to the specified account.
```

Comment:
I guess than when implementing this, all account members will be allowed to list account team members (since regular users are already [able to change release settings](https://athenianco.atlassian.net/browse/ENG-696?focusedCommentId=12352), I'd say that they should be also able to list team members) 
wdyt @warenlg ?

Reply:
yup as described in ENG-726 all users can manage teams.